bot/exts/help_channels/_cog.py

```diff
@@ -427,6 +430,21 @@ async def on_message(self, message: discord.Message) -> None:
         if channel_utils.is_in_category(message.channel, constants.Categories.help_available):
             if not _channel.is_excluded_channel(message.channel):
                 await self.claim_channel(message)
+                # Initialise the cache for this channel
+                await _caches.claimant_last_message_times.set(
+                    message.channel.id,
+                    message.created_at.timestamp()
+                )
+                await _caches.last_message_times.set(
+                    message.channel.id,
+                    message.created_at.timestamp()
+                )
+        elif channel_utils.is_in_category(message.channel, constants.Categories.help_in_use):
+            # Overwrite the claimant message time, if its from the claimant.
+            if message.author == await _caches.claimants.get(message.channel.id):
+                await _caches.claimant_last_message_times(message.channel.id, message.created_at.timestamp())
+
+            await _caches.last_message_times.set(message.channel.id, message.created_at.timestamp())
```

Comment:
This could be moved into `check_for_answer`  since it shares similar logic. In fact, I think you could get rid of `last_message_times` and instead repurpose the unanswered cache by storing times instead of booleans (where None would mean unanswered).